docs/user/metal/install_ipi.md

```diff
@@ -124,6 +124,8 @@ controlPlane:
     baremetal: {}
 platform:
   baremetal:
+    bootstrapMachineOSImage: https://mirror.example.com/path/to/rhcos-bootstrap-image.qcow.gz
+    clusterMachineOSImage: https://mirror.example.com/path/to/rhcos-baremetal-node-image.qcow.gz
```

Comment:
are these required values? if not let's drop them from default example.

Reply:
They are not required but I think mirroring these large images will be a common requirement - perhaps we can instead start a docs/user/metal/customization.md with a section showing these and other optional interfaces?

Reply:
> perhaps we can instead start a docs/user/metal/customization.md with a section showing these and other optional interfaces?

i'm fine with that. just make sure we make the distinction of baremetal-ipi and none (baremetal)